website/src/pages/flu/h5n1/single-variant.astro

```diff
@@ -33,80 +29,81 @@ const initialMutations = getLapisMutations(pageState.variantFilter);
 <OrganismPageLayout view={view}>
     <gs-app lapis={getLapisUrl(Organisms.h5n1)}>
         <div class='grid-cols-[300px_1fr] gap-x-4 sm:grid'>
-            <div>
-                <div class='bg-blue-50 px-2 py-4'>
-                    <div class='mb-2 font-semibold'>Filter dataset:</div>
-                    <LocationTimeFilter
-                        fields={view.locationFields}
-                        initialLocation={pageState.baselineFilter.location}
-                        initialDateRange={pageState.baselineFilter.dateRange}
-                        earliestDate={view.earliestDate}
-                        customDateRangeOptions={view.customDateRangeOptions}
-                        organismViewKey={organismViewKey}
-                    />
-                </div>
-                <div class='border-t-2 bg-green-50 px-2 py-4'>
-                    <div class='mb-2 font-semibold'>Search variant:</div>
-                    <VariantFilter fields={[]} initialMutations={initialMutations} organismViewKey={organismViewKey} />
-                </div>
-            </div>
-            <div>
-                <ComponentsGrid>
```

Comment:
Does removing this div break the grid layout?

This is how it looks now: 
![grafik](https://github.com/user-attachments/assets/61bb40f7-8eff-459b-b7c0-52bf5011aa7c)

This is how it should look like:
![grafik](https://github.com/user-attachments/assets/5e61471c-4bd2-4b39-a8d8-6ea17c55c216)

Reply:
I changed it, so the mutations over time components are again each on a single row.